angular/src/app/utilities/permissions.ts

```diff
@@ -0,0 +1,10 @@
+export default function validatePermissions(roleArray) {

```

Comment:
Are you using this anywhere? We're already handling frontend permission with OAuth